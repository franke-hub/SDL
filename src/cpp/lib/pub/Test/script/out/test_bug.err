This appears in TRACE and STDERR
??????????.??? This appears in TRACE and STDERR
Ignore mode: This appears in STDERR (even with ignore mode): *
This appears in TRACE and STDERR
??????????.??? This appears in TRACE and STDERR
