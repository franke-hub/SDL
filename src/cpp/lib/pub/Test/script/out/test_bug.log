Backtrace test
[bt]  0 * at /home/eskesen/obj/cpp/lib/pub/Test/S/Test_bug.cpp:*
[bt]  1 * at /home/eskesen/obj/cpp/lib/pub/Test/S/Test_bug.cpp:*
[bt]  2 * at /home/eskesen/obj/cpp/lib/pub/S/Wrapper.cpp:*
[bt]  3 * at /home/eskesen/obj/cpp/lib/pub/Test/S/Test_bug.cpp:*
*

Standard mode:
This appears in TRACE and STDOUT
This appears in TRACE and STDERR
This appears in TRACE ONLY
??????????.??? This appears in TRACE and STDOUT
??????????.??? This appears in TRACE and STDERR
??????????.??? This appears in TRACE ONLY
Intensive mode:
This appears in TRACE and STDOUT
This appears in TRACE and STDERR
This appears in TRACE ONLY
??????????.??? This appears in TRACE and STDOUT
??????????.??? This appears in TRACE and STDERR
??????????.??? This appears in TRACE ONLY
