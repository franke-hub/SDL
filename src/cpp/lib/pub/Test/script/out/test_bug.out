Backtrace test
[bt]  0 * at /home/eskesen/obj/cpp/lib/pub/Test/S/Test_bug.cpp:*
[bt]  1 * at /home/eskesen/obj/cpp/lib/pub/Test/S/Test_bug.cpp:*
[bt]  2 * at /home/eskesen/obj/cpp/lib/pub/S/Wrapper.cpp:*
[bt]  3 * at /home/eskesen/obj/cpp/lib/pub/Test/S/Test_bug.cpp:*
*

Standard mode:
This appears in TRACE and STDOUT
??????????.??? This appears in TRACE and STDOUT
Intensive mode:
This appears in TRACE and STDOUT
??????????.??? This appears in TRACE and STDOUT
